vividus-extension-selenium/src/main/java/org/vividus/ui/action/search/ElementActionService.java

```diff
@@ -20,16 +20,18 @@
 import java.util.Set;
 import java.util.stream.Collectors;
 
+import javax.inject.Inject;
+
 public class ElementActionService
 {
-    private final Set<IElementAction> elementActions;
+    @Inject private Set<IElementAction> elementActions;
 
-    private final Set<LocatorType> searchLocatorTypes;
-    private final Set<LocatorType> filterLocatorTypes;
+    private Set<LocatorType> searchLocatorTypes;
+    private Set<LocatorType> filterLocatorTypes;
 
-    public ElementActionService(Set<IElementAction> elementActions)
+    @SuppressWarnings("unused")
+    private void init()
```

Comment:
I recommend to keep it `public` since `private` makes this class unusable outside Spring